test/rdf-insert-mocha.js

```diff
@@ -0,0 +1,85 @@
+// rdf-insert-mocha.js
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.should();
+chai.use(chaiAsPromised);
+
+var http = require('http');
+var _ = require('underscore');
+var path = require('path');
+var noflo = require('noflo');
+var test = require('./common-test');
+var rdfLoad = require('../components/rdf-load');
+var rdfNtriples = require('../components/rdf-ntriples');
+var requestTemplate = require('../components/request-template');
+
+describe('rdf-insert subgraph', function() {
+    var server = http.createServer();
+    server.on('request', function(req, res) {
+        var body = [];
+        req.on('data', function(chunk) {
+            body.push(chunk);
+        }).on('end', function() {
+            body = Buffer.concat(body).toString();
+            res.end(body);
+        });
+    });
+    before(function(){
+        server.listen(1337);
```

Comment:
It could have been anything, I choose this number bc it wasn't 8080, which is more commonly used for services.